### PR TITLE
README.md update for the 1.13 release

### PR DESCRIPTION
**Pull request by hrydgard**
I'm gearing up to do a new release of PPSSPP soon, so here's the start of the README update.

As usual I'm sure I'm missing a ton of things, so feel free to contribute updates in the form of additional lines below.

### CODE REVIEW

**Comment by unknownbrackets** [comment]
Added a couple more.

-[Unknown]

**Comment by LunaMoo** [comment]
It's GOW:Ghost of Sparta, but pretty sure anyone interested will know that as the vortex stage is only in one of those games and the problem accumulated a lot of videos and people giving advices to change emulated psp cpu clock for it, which is not needed anymore(and was never a proper way to deal with it).
 Current solution isn't perfect either, since it works around a problem that would not happen if no hacks were used for this game at all, however this is better for end user as the game runs much better when limiting it's FPS to max it can display, better both for low end devices AND powerful PC's since the game simply runs smoother.

 CWCheat FPS patches for all GOW games also became obsolete at the same time as PPSSPP includes it's own version of that hack now which works better(CWCheat versions dropped a lot of FPS in certain stages like the vortex, PPSSPP solution doesn't suffer from that). Even includes 30 fps version available by editing compat.ini.

**Comment by iota97** [comment]
A couple more:

```
Added search in settings (#14414)
Added fast button reapeats on custom touch buttons (#15613)
Two new bicubic upscaling shader: Catmull-Rom and Mitchell-Netravali (#15569)
Allow to change screen rotation per game and to bind a key to change it (#15494, #15510)
Restored software rendering option on Android (#12958)
```

**Comment by LunaMoo** [comment]
Just to note something broke with DLC decryption or loading, I have no time to test it properly now, but might consider delaying new release - https://forums.ppsspp.org/showthread.php?tid=28800

**Comment by hrydgard** [comment]
That does sound like a regression - bisecting down to one commit would be very helpful.

**Comment by Felipefpl** [comment]
@LunaMoo - I see, thanks for the info. :+1:

**Comment by Gamemulatorer** [comment]
Workaround Star Wars Clone Wars and Force Unleashed graphics issue.
https://github.com/hrydgard/ppsspp/pull/15691
https://github.com/hrydgard/ppsspp/pull/15696

**Comment by iota97** [comment]
I feel like the sections separation is a bit off, things like "search for setting" should belong to UI imho. Not sure where to draw a line between general and UI tbh...

**Comment by hrydgard** [comment]
Yeah, not easy to draw those lines. Will try to reorganize a bit soon.

**Comment by LunaMoo** [comment]
Into game fixes also:
````
* Zettai Zetsumei Toshi 3 no longer hangs on character select screen ([#15687])
[#15687]: https://github.com/hrydgard/ppsspp/issues/15687 "Add Zettai Zetsumei Toshi 3 bypass for softlock"
````

File: README.md [hunk 1]
```diff
@@ -25,6 +25,49 @@ If you want to download regularly updated builds for Android, Windows x86 and x6
...
+* Allow to change screen rotation per game and to bind a key to change it ([#15494], [#15510])
+* Re-enabled software rendering option on Android ([#12958])
+
+Game fixes
+
+* Add more workarounds for Mali driver bugs ([#15016])
+* Vortex in God of War: Ghost of Sparta can now be passed ([#15640])
+* Various proAdhoc fixes ([#15213]), ([#15215]), and many more
+* Correct flickering text in Sol Trigger and Last Ranker. ([#15549])
+* Fix and improve line drawing in Echochrome ([#15583]), after line refactoring ([#15073]), ([#15075]), ([#15075])
```
Comment (iota97) [inline README.md:57]: I think Saramagrean was talking about this line too as 15075 appears twice in the list.

Comment (Saramagrean) [reply README.md:57]: and Background loading of texture replacement is should https://github.com/hrydgard/ppsspp/pull/15025 ,right?

Comment (hrydgard) [reply README.md:57]: Yeah, that's right, I'll fix both things.

**Pull request merged**